infra/src/dsl/xroad.ts

```diff
@@ -118,7 +118,7 @@ export const DrivingLicense = new XroadConf({
       prod: 'r1/IS/GOV/5309672079/Logreglan-Protected/Okuskirteini-v1',
     },
     XROAD_DRIVING_LICENSE_V5_PATH: {
-      dev: 'r1/IS-DEV/GOV/10005/Logreglan-Protected/Okuskirteini-v5',
+      dev: 'r1/IS-DEV/GOV/10005/Logreglan-Protected/okuskirteini-v5',
```

Comment:
When connecting to the dev server, it's 'okuskirteini-v5' instead of 'Okuskirteini-v5' for some reason.
This fixes it quickly though a case could be made to contact the relevant parties and ask them to change it to a consistant value.
This feels low priority though.